cpmonitor/models.py

```diff
@@ -474,6 +474,168 @@ class Meta:
     )
 
 
+class EnergyPlanChecklist(models.Model):
+    class Meta:
+        verbose_name = "Wärmeplanung Checkliste"
+
+    city = models.OneToOneField(
+        City, on_delete=models.PROTECT, related_name="energy_plan_checklist"
+    )
+
+    energy_plan_exists = models.BooleanField(
+        "Liegt ein öffentlich bekannt gemachter Beschluss zur Durchführung der Wärmeplanung vor?",
+        default=False,
+    )
+    energy_plan_exists_rationale = models.TextField(
+        "Begründung",
+        blank=True,
+    )
+    schedule_exists = models.BooleanField(
+        "Enthält der Beschluss einen Zeitplan für die Durchführung der Wärmeplanung (Ausschreibung, Beauftragung, Durchführung)?",
+        default=False,
+    )
+    schedule_exists_rationale = models.TextField(
+        "Begründung",
+        blank=True,
+    )
+    hydrogen_grid_examined = models.BooleanField(
+        "Wurde ein frühzeitiger Ausschluss von Wasserstoffnetzen geprüft und das Ergebnis der Prüfung begründet?",
+        default=False,
+    )
+    hydrogen_grid_examined_rationale = models.TextField(
+        "Begründung",
+        blank=True,
+    )
+    thermal_grid_examined = models.BooleanField(
+        "Wurde ein frühzeitiger Ausschluss von Wärmenetzen geprüft und das Ergebnis der Prüfung begründet?",
+        default=False,
+    )
+    thermal_grid_examined_rationale = models.TextField(
+        "Begründung",
+        blank=True,
+    )
+    demand_specified = models.BooleanField(
+        "Ergibt sich ein klares gebäudescharfes Bild des Wärmebedarfs und der aktuellen Wärmeversorgungsart?",
+        default=False,
+    )
```

Comment:
The other checklists have help_texts. Does it make sense here as well?

Reply:
I would leave it for now. Probably there will be anyway some adaptations to the layout and texts and then I'd rather first have some prototype that people can look at.